src/NServiceBus.Core.Tests/Routing/MessageDrivenSubscriptions/MessageDrivenSubscriptionsConfigExtensionsTests.cs

```diff
@@ -0,0 +1,182 @@
+﻿namespace NServiceBus.Core.Tests.Routing.MessageDrivenSubscriptions
+{
+    using System;
+    using System.Linq;
+    using System.Reflection;
+    using EventNamespace;
+    using MessageNameSpace;
+    using NServiceBus.Features;
+    using NServiceBus.Routing;
+    using NServiceBus.Routing.MessageDrivenSubscriptions;
+    using NUnit.Framework;
+    using OtherMesagenameSpace;
+    using Settings;
+    using Transport;
+
+    [TestFixture]
+    public class MessageDrivenSubscriptionsConfigExtensionsTests
```

Comment:
(merged tests from this branch with those from `MessageDrivenSubscriptionsConfigExtensionsTests` in the general routing namespace into this test class)